### DISCUSSION TITLE
Install script not finishing and plugnmeet-plugnmeet-1 not starting

### DISCUSSION THREAD

**Discussion by yasmagic** [mynaparrot/plugNmeet-server#605] [General]
When installing using install script, plugnmeet-plugnmeet-1 container not starting. See screenshot below. How to resolve this? I cannot login to the container as it is always in 'starting' phase and cannot view logs as well.

![image](https://github.com/user-attachments/assets/a376f859-4a2e-46f0-a611-a08a5fd46e72)

**Comment by yasmagic** [comment]
Log says:

{
  "file": "app/main.go:54",
  "func": "main.startServer",
  "level": "fatal",
  "msg": "nats: Authorization Violation",
  "time": "2024-12-04T18:51:35Z"
}

**Reply by mpakzadm2** [reply]
nat-server,conf  user/password is empty 
bug in install.sh

**Reply by yasmagic** [reply]
Thank you. Please provide fixed install.sh.

**Comment by jibon57** [comment]
Fix: https://github.com/mynaparrot/plugNmeet-install/commit/8dc5471764d2ad4e775e1143ddbdde1410ba9d0d

**Comment by yasmagic** [comment]
It worked, thank you
